daemon/snap.go

```diff
@@ -134,11 +143,15 @@ func allLocalSnapInfos(st *state.State, all bool, wanted map[string]bool) ([]abo
 		if err != nil {
 			return nil, err
 		}
+
 		refreshInhibitProceedTime := snapst.RefreshInhibitProceedTime(st)
+		if sel == snapSelectRefreshInhibited && refreshInhibitProceedTime.IsZero() {
```

Comment:
can this timestamp effectively be in the past wrt to time.Now()? I suppose so, given that it's cleared in link-current, so `select=refresh-inhibited` effectively means select snaps which may have had their refresh inhibited at some point. I'm asking in case the intention was to list only snaps which inhibited at the time of the query.

Reply:
I suppose what I'm asking is whether we need to check refreshInhibitProceedTime.Before(time.Now()) rather than IsZero()

Reply:
I think the condition should probably be something like either proceed-time is before now or the snap is being monitored (which means that is still the case that closing it would trigger a refresh)